diff --git a/docs/config.md b/docs/config.md
index 9831b9f..300e80e 100644
--- a/docs/config.md
+++ b/docs/config.md
@@ -27,6 +27,7 @@ dir or use the `--config` option to specify the location of your configuration f
 - **Skip Tag:** Skip automatic commit tagging
 - **Skip Verify:** Skip the pre-commit and commit-msg hooks
 - **Disable Links:** Render text instead of link in changelog
+- **Changelog Version Format:** Allows the version header in changelog to have a configurable format
 - **Hidden Hash:** Hide commit hash from changelog
 - **Hidden Mentions:** Hide users mentions from changelog
 - **Hidden References:** Hide issue references from changelog
@@ -84,6 +85,7 @@ return [
   'skipTag' => false,
   'skipVerify' => false,
   'disableLinks' => false,
+  'changelogVersionFormat' => '## {{version}} ({{date}})',
   'hiddenHash' => false,
   'hiddenMentions' => false,
   'hiddenReferences' => false,
diff --git a/src/Changelog.php b/src/Changelog.php
index 2e5aa4e..8576ec8 100644
--- a/src/Changelog.php
+++ b/src/Changelog.php
@@ -351,7 +351,8 @@ class Changelog
             $params['to'] = $this->getVersionCode($params['to'], $tagPrefix, $tagSuffix);
             $compareUrl = $this->getCompareUrl($params['from'], "{$tagPrefix}{$params['to']}{$tagSuffix}");
             $markdownCompareLink = $this->getMarkdownLink($params['to'], $compareUrl);
-            $changelogNew .= "## {$markdownCompareLink} ({$params['date']})\n\n";
+            $changeLogVersionHeading = $this->getChangelogVersionHeading($markdownCompareLink, $params['date']);
+            $changelogNew .= $changeLogVersionHeading;
             // Add all changes list to new changelog
             $changelogNew .= $this->getMarkdownChanges($changes);
         }
@@ -546,6 +547,19 @@ class Changelog
         return !$this->config->isDisableLinks() ? "[{$text}]({$url})" : $text;
     }
 
+    protected function getChangelogVersionHeading($markdownCompareLink, $versionDate)
+    {
+        $versionFormat = $this->config->getChangelogVersionFormat();
+
+        // Handle the replacements.
+        $versionData = $this->getCompiledString($versionFormat, [
+            'version' => $markdownCompareLink,
+            'date' => $versionDate,
+        ]);
+
+        return $versionData;
+    }
+
     /**
      * Compile string with values.
      */
diff --git a/src/Configuration.php b/src/Configuration.php
index ec32ab7..10e8c01 100644
--- a/src/Configuration.php
+++ b/src/Configuration.php
@@ -138,6 +138,13 @@ class Configuration
      */
     protected $disableLinks = false;
 
+    /**
+     * Allows configurable changelog version header format.
+     *
+     * @var string
+     */
+    protected $changelogVersionFormat = '## {{version}} ({{date}})';
+
     /**
      * Hidden references.
      *
@@ -294,6 +301,7 @@ class Configuration
             'skipTag' => $this->skipTag(),
             'skipVerify' => $this->skipVerify(),
             'disableLinks' => $this->isDisableLinks(),
+            'changelogVersionFormat' => $this->getChangelogVersionFormat(),
             'hiddenHash' => $this->isHiddenHash(),
             'hiddenMentions' => $this->isHiddenMentions(),
             'hiddenReferences' => $this->isHiddenReferences(),
@@ -370,6 +378,8 @@ class Configuration
             ->setIssueUrlFormat($params['issueUrlFormat'])
             ->setUserUrlFormat($params['userUrlFormat'])
             ->setReleaseCommitMessageFormat($params['releaseCommitMessageFormat'])
+            ->setDisableLinks($params['disableLinks'])
+            ->setChangelogVersionFormat($params['changelogVersionFormat'])
             // Hooks
             ->setPreRun($params['preRun'])
             ->setPostRun($params['postRun']);
@@ -693,6 +703,18 @@ class Configuration
         return $this;
     }
 
+    public function getChangelogVersionFormat(): string
+    {
+        return $this->changelogVersionFormat;
+    }
+
+    public function setChangelogVersionFormat($changelogVersionFormat): self
+    {
+        $this->changelogVersionFormat = $changelogVersionFormat;
+
+        return $this;
+    }
+
     public function isHiddenReferences(): bool
     {
         return $this->hiddenReferences;
